src/device-registry/controllers/create-activity.js

```diff
@@ -15,11 +15,15 @@ const activity = {
       const hasErrors = !validationResult(req).isEmpty();
       if (hasErrors) {
         let nestedErrors = validationResult(req).errors[0].nestedErrors;
-        logger.error(
-          `input validation errors ${errors.convertErrorArrayToObject(
-            nestedErrors
-          )}`
-        );
+        try {
+          logger.error(
+            `input validation errors ${JSON.stringify(
```

Comment:
just stringifying the error messages to ensure that they are readable in the corresponding Slack notifications.

Reply:
Doing this throughout the PR and also adding some new error logging for more implementations as I handle exceptions accordingly.